datadog_lambda/wrapper.py

```diff
@@ -57,9 +57,8 @@ def __init__(self, func):
         patch_all()
         logger.debug("datadog_lambda_wrapper initialized")
 
-    def _before(self, event, context):
+    def _before(self, event, context, **kwargs):
```

Comment:
Is it necessary to pass `kwargs` to `_before()` and `_after()`? I don't think any of the datadog logic depends (or should depend) on the `kwargs`.

Reply:
Makes sense. Gonna remove that.